add-ons/fabric8/fabric8.addon

```diff
@@ -0,0 +1,37 @@
+# Name: fabric8
+# Description: Installs fabric8 locally on Minishift
+# Required-Vars: GITHUB_OAUTH_CLIENT_ID,GITHUB_OAUTH_CLIENT_SECRET
```

Comment:
no, this is different. These are not the GitHub API tokens. This is about Fabric8 and getting OAuth to work. At least that's my understanding